libretroshare/src/grouter/p3grouter.cc

```diff
@@ -3,7 +3,8 @@
  *                                                                             *
  * libretroshare: retroshare core library                                      *
  *                                                                             *
- * Copyright 2013 by Cyril Soler <csoler@users.sourceforge.net>                *
+ * Copyright (C) 2013  Cyril Soler <csoler@users.sourceforge.net>              *
+ * Copyright (C) 2019  Gioacchino Mazzurco <gio@eigenlab.org>                  *
```

Comment:
Personnally I dont add my own name everytime I modify a file, unless for a full rewrite, which means the file can be considered as new. For small changes, individual commits hold your name already and git blame give the exact spectrum of your contributions.